src/test/java/chess/BoardTest.java

```diff
@@ -0,0 +1,31 @@
+package chess;
+
+import chess.pieces.Pawn;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class BoardTest {
+
+    Pawn white, black;
```

Comment:
한 줄에 하나의 변수만 선언하는 것을 원칙으로 해 주세요. 

```suggestion
    private Pawn white;
    private Pawn black;
```

Reply:
네에! 앞으로 한줄에 하나의 변수만 선언하도록 하겠습니다